src/os/bluestore/BlueFS.cc

```diff
@@ -546,6 +546,13 @@ uint64_t BlueFS::get_block_device_size(unsigned id) const
   return 0;
 }
 
+BlockDevice* BlueFS::get_block_device(unsigned id) const
+{
+  if (id < bdev.size() && bdev[id])
```

Comment:
nit: "&& bdev[i]" is redundant.

Reply:
True, but existence of the check highlights that we return `null` when out of range or bdev not created.